File: Parte_1/codigos/micro09.py
def micro09():

	preco = 0.0
    venda = 0.0
    novopreco = 0.0

	print("Digite o preco: ",end = "")
	preco = int(raw_input())
	print("Digite a venda: ",end = "")
	venda = int(raw_input())
	if venda < 500 or preco <30:
		novopreco = preco + 10/100 *preco
	elif (venda >= 500 and venda <1200) or (preco >= 30 and preco <80):
		novopreco = preco + 15/100 * preco
	elif venda >=1200 or preco >=80:
		novopreco = preco - 20/100 * preco

	print("O novo preco e: "+str(novopreco))

micro09()

File: Parte_1/codigos/micro09.c
#include <stdio.h>

int main(){
	
	float preco, venda, novo_preco;

	printf("Digite o preco: ");
	scanf("%f", &preco);
	printf("Digite a venda: ");
	scanf("%f", &venda);

	if ( (venda <500) || (preco < 30)){
		novo_preco = preco + (10.0/100.0 * preco);
	}
	else if( (venda >= 500 && venda < 1200) || (preco >= 30 && preco < 80)){ 
		novo_preco = preco + (15/100 * preco);
	}
	else if(venda >= 1200 || preco >= 80){
		novo_preco = preco - (20/100 * preco);
	}

	printf("O novo preco é %f", novo_preco);

}

